frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/UsersPage/UsersPage.tsx

```diff
@@ -122,6 +133,18 @@ const UsersPage = ({ location, router }: ITeamSubnavProps): JSX.Element => {
     [teams, teamIdForApi]
   );
 
+  useEffect(() => {
+    setTeamUsersState(() => {
+      return (
+        teamUsers?.filter((user) => {
+          return user.name
+            .toLowerCase()
+            .includes(usersSearchString.toLowerCase());
+        }) || []
+      );
+    });
+  }, [usersSearchString, teamUsers]);
```

Comment:
Do we need `teamUserState` to be managed by react state and does this need to be in a `useEffect`? Can we calculate teamUsersState when we get teamUsers?

something like:

```tsx
const teamUserState = teamUsers?.filter((user) => {
  return user.name
    .toLowerCase()
    .includes(usersSearchString.toLowerCase());
  }) || []
```

Reply:
Oh nice catch thank you! This is server side filter so updating now!